website_and_docs/content/documentation/webdriver/elements/information.zh-cn.md

```diff
@@ -6,23 +6,13 @@ description: >
   元素相关的知识.
 ---
 
-您可以查询有关特定元素的许多详细信息.
+您可以查询有关特定元素的许多详细信息。
```

Comment:
```suggestion
您可以查询有关特定元素的许多详细信息.
```

Reply:
For those punctuation marks, no worries, no need to change back again. I won't change other translators' detail points if no content impacting.

Please keep your content following doc rule is fine, at least "Wrap all other lines at 72 columns."